### PR TITLE
Added feature to Bepis to get random ㄒ卄乇爪乇爪乙Ҝ丨几Ꮆ quotes

### PR DESCRIPTION
**Pull request by Tmpecho**
Probably something wrong with using the csv file. Does not include quotes from before update is added. Please go through the thememzking_collector.py file and check for errors as I can't test run on Discord.

### CODE REVIEW

File: thememzking_collector.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import discord
+import pandas as pd
```
Comment (soitchy) [inline thememzking_collector.py]: Unused import?

File: thememzking_collector.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+quotes_file = open("lhh_discord/thememzking_quotes.csv", "a")
+
+thememzking_id = 410169704338948096
+
+client = discord.Client()
+
+@client.event
+async def on_message(msg):
+    if msg.author.id == thememzking_id:  # if from thememzking
+        await add_to_df(msg)
```
Comment (soitchy) [inline thememzking_collector.py]: does add_to_df need to be awaited?

File: thememzking_collector.py (hunk 1 above)
Comment (Tmpecho) [reply thememzking_collector.py]: Yes I first used pandas but removed it without removing the import. Forgot that I didn't need to import discord.

File: thememzking_collector.py (hunk 2 above)
Comment (Tmpecho) [reply thememzking_collector.py]: I'm not 100% sure. Maybe not.

File: thememzking_collector.py (hunk 1 above)
Comment (Tmpecho) [reply thememzking_collector.py]: I have to use import discord to get new messages. I updated the file to remove the pandas import and the awaiting. Unsure on how to update my pull request :P

**Pull request closed without merging**